src/elements/RFCavity.H

```diff
@@ -89,17 +89,17 @@ namespace impactx::elements
 namespace RFCavityData
 {
     //! last used id for a created RF cavity
-    static inline int next_id = 0;
```

Comment:
`static inline` is for class member variables. Here we are global vars in namespaces.